docs/howto/openstack/nova/boot-image-volume.md

```diff
@@ -0,0 +1,405 @@
+# Converting boot-from-image servers to boot-from-volume
+
+You can freely move any server [between {{brand}}
+regions](move-server-between-regions), provided it boots from a volume
+(which we generally recommend since it affords more flexibility than
+booting from an image). You may still have boot-from-image servers,
+though. To ensure a particular server is of that type, you can go to the
+{{gui}}, locate the server, and expand its detailed view. Go to the
+*Details* tab and pay attention to the *Attached volumes* field. If this
+is a boot-from-image server, it should say *No attached volumes*.
```

Comment:
Hmmm is this a reliable test? What if it's a boot-from-image server that has had a (non-boot) volume attached after spinning up?

Isn't looking the prior field, "Boot Target", a better option here?

Reply:
It sure is :)